### PR TITLE
Add auto-generated headers to CS files to avoid code analysis warnings.

### PR DESCRIPTION
**Pull request by jlaanstra**
Add the standard ```<auto-generated />``` header to avoid code analysis warnings when the files are included.

### CODE REVIEW

**Comment by jlaanstra** [comment]
Text in between auto-generated tags can be customized.

**Review by BenJKuhn — APPROVED** [review]

File: dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs [hunk 1]
```diff
@@ -1,4 +1,13 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// ------------------------------------------------------------------------------
+// <auto-generated>
```
Comment (DrusTheAxe) [inline dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs:2]: This is NOT a generated file. It's artisanal hand-crafted code

File: dev/UndockedRegFreeWinRT/UndockedRegFreeWinRT-AutoInitializer.cs [hunk 2]
```diff
@@ -1,3 +1,12 @@
+// ------------------------------------------------------------------------------
+// <auto-generated>
```
Comment (DrusTheAxe) [inline dev/UndockedRegFreeWinRT/UndockedRegFreeWinRT-AutoInitializer.cs:2]: This is NOT a generated file. It's artisanal hand-crafted code

File: dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cs [hunk 3]
```diff
@@ -1,3 +1,12 @@
+// ------------------------------------------------------------------------------
+// <auto-generated>
```
Comment (DrusTheAxe) [inline dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cs:2]: This is NOT a generated file. It's artisanal hand-crafted code

**Review by DrusTheAxe — CHANGES_REQUESTED** [review]
These files are NOT generated

What's the intent here?

**Comment by DrusTheAxe** [comment]
> Add the standard `<auto-generated />` header to avoid code analysis warnings when the files are included.

I can't find a good explanation about `<auto-generated />`. Got link?

File: dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs (hunk 1 above)
Comment (riverar) [reply dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs:2]: It is said a robot wrote this code. <g>

File: dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cs (hunk 3 above)
Comment (BenJKuhn) [reply dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cs:2]: It's generated in the sense that it's copied into someone else's project, and they have no ability to modify it.

**Comment by jlaanstra** [comment]
> > Add the standard `<auto-generated />` header to avoid code analysis warnings when the files are included.
> 
> I can't find a good explanation about `<auto-generated />`. Got link?

Docs are slim, closest I've found is this: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1145#issuecomment-128672540

**Comment by jlaanstra** [comment]
> These files are NOT generated
> 
> What's the intent here?

The intent is for me to be able to enable my code style and enforce it with treatwarningsaserrors without it complaining about incuded files from WInAppSDK that I can't modify.

File: dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs (hunk 1 above)
Comment (DrusTheAxe) [reply dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs:2]: > It is said a robot wrote this code. `<g>`

I've been called many things (some of them even printable) but first time a robot :P

**Comment by DrusTheAxe** [comment]
> > These files are NOT generated
> > What's the intent here?
> 
> The intent is for me to be able to enable my code style and enforce it with treatwarningsaserrors without it complaining about incuded files from WInAppSDK that I can't modify.

OK that makes sense. Thanks for the suggestion and the link

I couldn't update this PR with revised wording so I made a parallel PR as a revised form of this one
https://github.com/microsoft/WindowsAppSDK/pull/2533

**Pull request closed without merging**